### ISSUE TITLE
Proper error checking instead of @

### ISSUE THREAD

**Issue by emente** [datto/php-json-rpc#5]
Server.php/147:
```
        $version = @$request['jsonrpc'];
        if (@$version !== self::VERSION) {
        $method = @$request['method'];
```

Because on most dev servers it results in the following:
> SCREAM:  Error suppression ignored for
> Notice: Undefined index: jsonrpc in /var/www/foo/vendor/datto/json-rpc/src/Server.php on line 147

**Comment by smortensen** [comment]
Okay, I've released version 4.0.3, which eliminates the error suppression. Does this work for you?

**Comment by mar-tin-f** [comment]
Yes, works. Perfect. Thank you.
